tools/doc/type-parser.js

```diff
@@ -38,8 +43,9 @@ module.exports = {
       typeText = typeText.trim();
       if (typeText) {
         let typeUrl = null;
-        if (jsPrimitives.indexOf(typeText) !== -1) {
-          typeUrl = jsPrimitiveUrl + '#' + typeText + '_type';
+        const primitive = jsPrimitives[typeText];
+        if (primitive !== undefined) {
```

Comment:
Any reason for explicitly checking for `!== undefined`? Isn't `if (primitive)` sufficient?